src/arrayMethodSort.js

```diff
@@ -3,9 +3,44 @@
 /**
  * Implement method Sort
  */
+
+const compareAsString = (a, b) => {
+  const aToString = String(a);
+  const bToString = String(b);
+
+  switch (true) {
+    case aToString > bToString:
+      return 1;
+
+    case aToString === bToString:
+      return 0;
+
+    default:
+      return -1;
+  }
```

Comment:
This use of `switch/case` is valid, but not very appropriate. Rewrite using `if/return` structure.